util/uvmdvgen/index.md.tpl

```diff
@@ -120,7 +120,7 @@ Please take a look at the link for detailed information on the usage, capabiliti
 Here's how to run a basic sanity test:
 ```console
 $ cd hw/ip/${name}/dv
-$ make TEST_NAME=${name}_sanity
+$ ../../../util/dvsim/dvsim.py ${name}_sim_cfg.hjson -i ${name}_sanity
```

Comment:
Probably better to ref the command from $REPO_TOP.

Reply:
Sounds good, thanks Sri I will update that.